src/app/pages/common/ix-forms/components/ix-input-password/ix-input-password.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  ChangeDetectionStrategy, ChangeDetectorRef, Component, Input,
+} from '@angular/core';
+import { ControlValueAccessor, FormControl, NgControl } from '@angular/forms';
+import { UntilDestroy } from '@ngneat/until-destroy';
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-input-password',
+  templateUrl: './ix-input-password.component.html',
+  styleUrls: ['./ix-input-password.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class IxInputPasswordComponent implements ControlValueAccessor {
+  @Input() label: string;
+  @Input() placeholder: string;
+  @Input() prefixIcon: string;
```

Comment:
With such a specific input, we can remove such fields which we know we won't be using.

Reply:
What if we wish to refactor the sign-in form and want to use this component?